src/app/jscaip/tests/Coord.spec.ts

```diff
@@ -136,4 +136,27 @@ describe('Coord', () => {
 
     });
 
+    describe('getDistanceToward', () => {
+
+        const coord: Coord = new Coord(0, 0);
+
+        it('should calculate result and not check alignment', () => {
+            const distance: number = coord.getDistanceToward(new Coord(1, 1), false);
+            expect(distance).toBe(1);
+        });
+
+        it('should calculate result for "knight-like" distance without checking alignment', () => {
+            const distance: number = coord.getDistanceToward(new Coord(1, 2), false);
+            expect(distance).toBe(2);
+        });
+
+        it('should throw and check alignement', () => {
```

Comment:
should throw when checking alignment and coords are not aligned

Reply:
true is not aligned !